src/client/preview/client_api.js

```diff
@@ -24,6 +24,10 @@ export default class ClientApi {
   }
 
   storiesOf(kind, m) {
+    if (!kind) {
+      throw "Invalid kind provided for stories";
```

Comment:
The build's failing because of a couple simple linting errors.

This should be `throw new Error('Invalid kind provided for stories');`

Also, remove the whitespace on line 30.